amqp/amqp.php

```diff
@@ -121,13 +121,22 @@
 /**
  *
  */
-define('AMQP_OS_SOCKET_TIMEOUT_ERRNO', 536870923);
+define('AMQP_OS_SOCKET_TIMEOUT_ERRNO', 536870947);
```

Comment:
One of tests fails due to this incorrect value. Please see https://github.com/pdezwart/php-amqp/blob/master/php_amqp.h for details on how this constant is calculated